roles/ezproxy/tasks/main.yml

```diff
@@ -52,6 +34,13 @@
   when:
     - running_on_server
 
+- name: Ezproxy | install ezproxy startup script (requires root)
+  ansible.builtin.command: /var/local/ezproxy/ezproxy -si
```

Comment:
Does this command create a file or install a service or something like? something we could use as a condition to make the command only run when it needs to? Then we wouldn't need the `changed_when: false` . . . or at the least we could set `changed_when:` to something more specific

Reply:
we get very little to go for what is does from [the documentation](https://help.oclc.org/Library_Management/EZproxy/Install_and_update_EZproxy/Install_and_update_self-hosted_EZproxy/EZproxy_for_Linux_install) :shrug: